content/docs/10-audit-logs/05-kube-api-audit-logging.md

```diff
@@ -0,0 +1,209 @@
+---
+title: "Enable Audit Logging"
+metaTitle: "Enable API Audit Logging"
+metaDescription: "Learn how to configure the kube-apiserver audit logging feature for Palette."
+icon: ""
+hideToC: true
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+import Tooltip from "shared/components/ui/Tooltip";
+
+# Enable Kubernetes API Audit Logging
+
+Kubernetes auditing is a feature of the Kubernetes cluster management system that allows administrators to track and log events within the cluster. Administrators can review actions taken by users and applications and changes to the cluster's configuration or state. By enabling auditing, organizations and system administrators can better understand their users' actions and behaviors. The audit log answers common questions about what, where, when, and who.
+
+You can also meet internal security control requirements by enabling audit logging. Many security controls require the following capabilities.
+
+- ensuring administrators can trace the actions of individual users back to a specific person.
+
+
+- to debug an issue where an unknown application is modifying resources
+
+The guidance on this page is based on the upstream Kubernetes documentation and `kube-apiserver` source code. Follow the steps below to enable audit logging for the Kubernetes API server.
+
+<br />
+
+<WarningBox>
+
+Enabling audit logging increases the memory consumption of the API server because some context required for auditing is stored for each request. Memory consumption depends on the audit logging configuration.
+
+</WarningBox>
+
+# Prerequisites
+
+- Access to a Kubernetes cluster node.
+
+
+- Write access to the file system.
+
+
+- Remote access to the Kubernetes cluster master nodes.
+
+
+
+# Enable Auditing
+
+The Kubernetes API Audit policies contain rules about what events are captured and the level of detail to include.
+The audit policy you create will capture all requests at the *metadata* level. To learn more about the various audit levels, visit the Kubernetes API [Audit Policy](https://kubernetes.io/docs/tasks/debug/debug-cluster/audit/#audit-policy) documentation.
+
+<br />
+
+1. Log in to [Palette](https://console.spectrocloud.com).
+
+
+2. Identify one of your cluster contro-plane nodes. You find a cluster node by navigating to the left **Main Menu** and selecting **Clusters**. Click on your cluster to access the details pages and click on the **Nodes** tab. The tab contains information about each pool, select a node from the **Master Pool** to view its IP address.
+
+
+3. SSH into one of your control-plane nodes by using its IP address and the SSH key you specified during the cluster creation process. 
```

Comment:
SSH into one of your control-plane nodes using its IP address and the SSH key you specified during the cluster creation process.